unfetter-discover/unfetter-ui
# Issue 630

This requires issue-630 from store.

https://github.com/unfetter-discover/unfetter-store/pull/75

Test plan:

- Requires two users in the same organization (Unfetter Open doesn't work)
- Create an analytic (in analytic hub) and verify that the other user is notified
- Verify that the link in the notification works
- Got to user settings (avatar dropdown in top right)
- Turn off subscriptions
- Make an analytic from the other use and confirm that there is NOT a notification
